README.md

```diff
@@ -144,3 +144,17 @@ chown root:root sedutil/Release_x86_64/sedutil-cli
 mv sedutil/Release_x86_64/sedutil-cli /usr/local/bin
 rm -rf ./sedutil*
 ```
+
+#### Test for Block SID Authentication
+
+Block SID: a mechanism by which a host application can alert the storage device
+to block attempts to authenticate the SID authority until a subsequent device power
+cycle occurs. This mechanism can be used by BIOS/platform firmware to prevent a
+malicious entity from taking ownership of a SID credential that is still set to
+its default value of MSID.
+
+- Fork of sedutil that supports Block SID: GitHub - [ChubbyAnt/sedutil](https://github.com/ChubbyAnt/sedutil)
```

Comment:
I'd simply say that the tests are written to use this fork until this is patched into the official build.

Reply:
thanks, updated